### PR TITLE
6 add tabs, display configurable list then non configurable

### PR DESCRIPTION
**Pull request by EgorEko**
Added sort_cubit. Display display configurable list.

### CODE REVIEW

File: example/lib/cubit/sort_cubit/sort_cubit.dart [hunk 1]
```diff
@@ -0,0 +1,390 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:native_filters/native_filters.dart';
+import 'package:rxdart/rxdart.dart';
+
+part 'sort_state.dart';
+
+class SortCubit extends Cubit<SortState> {
```
Comment (nikolaydymura) [inline example/lib/cubit/sort_cubit/sort_cubit.dart:8]: Rename to AvailableFiltersCubit

File: example/lib/cubit/sort_cubit/sort_cubit.dart [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+  SortCubit() : super(SortInitial());
+
+  @override
+  Stream<SortState> get stream => super.stream.doOnListen(() {
+        if (state is SortInitial) {
+          fetchSortData(const FilterFactory());
+        }
+      });
+
+  Future<void> fetchSortData(FilterFactory filtersFactory) async {
```
Comment (nikolaydymura) [inline example/lib/cubit/sort_cubit/sort_cubit.dart:18]: Init FiltersFactory into constructor

File: example/lib/list_filters.dart [hunk 3]
```diff
@@ -1,84 +1,42 @@
...
-          }
-          return const CircularProgressIndicator();
-        },
+    return DefaultTabController(
+      initialIndex: 0,
+      length: 2,
+      child: Scaffold(
+        appBar: AppBar(
+          title: Center(child: Text(widget.title)),
+          bottom: const TabBar(
```
Comment (nikolaydymura) [inline example/lib/list_filters.dart:21]: Add two more tabs , video and image (keep them empty for now)

File: example/lib/widgets/list_filters_widget.dart [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  final bool configurableFilters;
+
+  final filtersFactory = const FilterFactory();
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<SortCubit, SortState>(
+      builder: (context, state) {
+        if (state is SortInitial) {
+          context.read<SortCubit>().fetchSortData(const FilterFactory());
```
Comment (nikolaydymura) [inline example/lib/widgets/list_filters_widget.dart:23]: Remove loading data from here

File: lib/factory/filter_configuration_builder.dart [hunk 5]
```diff
@@ -0,0 +1,83 @@
+import 'dart:io';
```
Comment (nikolaydymura) [inline lib/factory/filter_configuration_builder.dart:1]: This file was merged in previous PR, remove it from this one

File: example/lib/widgets/list_filters_widget.dart [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+import '../cubit/sort_cubit/sort_cubit.dart';
+import '../filter_details.dart';
+
+class ListFiltersWidget extends StatelessWidget {
+  const ListFiltersWidget({
+    super.key,
+    required this.configurableFilters,
+  });
+
+  final bool configurableFilters;
```
Comment (nikolaydymura) [inline example/lib/widgets/list_filters_widget.dart:14]: Pass List<FilterItems> here

**Review by nikolaydymura — CHANGES_REQUESTED** [review]

File: example/lib/list_filters.dart (hunk 3 above)
Comment (EgorEko) [reply example/lib/list_filters.dart:21]: Added two new tabs.

File: example/lib/widgets/list_filters_widget.dart (hunk 4 above)
Comment (EgorEko) [reply example/lib/widgets/list_filters_widget.dart]: Removed loading data from here

File: example/lib/widgets/list_filters_widget.dart (hunk 6 above)
Comment (EgorEko) [reply example/lib/widgets/list_filters_widget.dart:14]: Сhanged to transmission List here

File: example/lib/cubit/sort_cubit/sort_cubit.dart (hunk 2 above)
Comment (EgorEko) [reply example/lib/cubit/sort_cubit/sort_cubit.dart]: Init FiltersFactory into main and pass List into cubit

File: lib/factory/filter_configuration_builder.dart (hunk 5 above)
Comment (EgorEko) [reply lib/factory/filter_configuration_builder.dart]: File was deleted from this PR

**Review by nikolaydymura — APPROVED** [review]

**Comment by EgorEko** [comment]
Close #9 added 4 tabs, display configurable list then non configurable (#9)

**Pull request merged**